lib/appsignal/event_formatter/rom/sql_formatter.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Appsignal
+  class EventFormatter
+    module Rom
+      class SqlFormatter
+        def format(payload)
+          ["query.#{payload[:name]}", payload[:query], SQL_BODY_FORMAT]
+        end
+      end
+    end
+  end
+end
+
+Appsignal::EventFormatter.register(
+  "sql.dry",
```

Comment:
But this has to match [the prefix given to look for an event formatter](https://github.com/appsignal/appsignal-ruby/pull/924/files/0fe4151b9c58b94020fc216ee7d34ab293cedc45#diff-0f8840148f218ced5733f91fd38db624eedd831c66a6511619386eba0be14e63R11) when the dry-monitor instrumentation receives the event -- and this instrumentation is generic to all dry-monitor events (just like the activesupport notifications one is generic to all activesupport events)

Reply:
If we do so, there's no way of automatically formatting the events `rom-sql` emits because they're called just `sql`. The suffix is added in the `dry-monitor` to add a minimum granularity.

I'm going to propose a change upstream to make `rom-sql` emit `sql.rom` events instead. As ActiveRecord does.